src/Chapter1.hs

```diff
@@ -503,8 +505,11 @@ branches because it is an expression and it must always return some value.
   satisfying the check will be returned and, therefore, evaluated.
 -}
 closestToZero :: Int -> Int -> Int
-closestToZero x y = error "closestToZero: not implemented!"
-
+closestToZero x y =
+  let
+    closest = min (abs x) (abs y)
+  in if closest == abs x then x else y
```

Comment:
Very interesting solution! 💡 

Though it could be a bit simplified. You see that all you need is to compare the absolute value of both `x` and `y`, and based on that return the result (as you do in the `then` and `else` clauses 👍🏼